### PR TITLE
Upates spawners with entity tables

### PR DESCRIPTION
**Pull request by IamVelcroboy**
## About the PR
- Updates Random Item, Device/Board, Random Box(and ammo) spawners with entity tables
- Adds Space Law book to Book spawner 
- Adjusts machine part spawners to indicate percentage since tiered parts no longer exist
- Add additional Ammo Box Spawner 90% for armories

## Why / Balance
Better balance, more organized, etc

## Technical details, ## Media, ## Breaking changes, **Changelog**
n/a-not player facing

### CODE REVIEW

File: Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml [hunk 1]
```diff
@@ -55,6 +55,7 @@
       - BookEngineersHandbook
       - BookScientistsGuidebook
       - BookSecurity
+      - BookSpaceLaw
```
Comment (deltanedas) [inline Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml]: doesnt this book not work

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml]: I'm not sure. I just saw that it was added in the last merge

Comment (deltanedas) [reply Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml]: upstream has a space law guidebook but we dont i think, thats why i had to comment out all the space law spawns

Comment (IamVelcroboy) [reply Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml]: I see. I'll comment it out when I fix conflicts. Maybe can be added back if we make our own then.

Comment (deltanedas) [reply Resources/Prototypes/Nyanotrasen/Entities/Markers/Spawners/Random/books.yml]: also the conflict is just from material reclaimer being removed

**Review by ghost — APPROVED** [review]
As far as I can tell this YAML seems to be fine. If the entire game explodes when I press the funny button, make me explode thanks 😸

**Pull request merged**